nautobot/extras/templates/extras/dynamicgroup.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'generic/object_detail.html' %}
+{% load helpers %}
+
+{% block content_left_page %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Dynamic Group</strong>
+        </div>
+        <table class="table table-hover panel-body attr-table">
+            <tr>
+                <td>Name</td>
+                <td><span>{{ object.name }}</span></td>
+            </tr>
+            <tr>
+                <td>Slug</td>
+                <td><span>{{ object.slug }}</span></td>
+            </tr>
+            <tr>
+                <td>Description</td>
+                <td><span>{{ object.description }}</span></td>
+            </tr>
+            <tr>
+                <td>Content Type</td>
+                <td><span>{{ object.content_type }}</span></td>
+            </tr>
+        </table>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Filter</strong>
+        </div>
+        <table class="table table-hover panel-body attr-table">
+            {% for param, value in object.filter.items %}
+                <tr>
+                    <td><code>{{ param }}</code></td>
+                    <td><code>{{ value }}</code></td>
+                </tr>
+            {% endfor %}
```

Comment:
Wonder if it'd be simpler (and maybe even cleaner?) to just do:

```jinja2
<pre>{{ object.filter | render_json }}</pre>

Reply:
I'm okay with that. Altho I do wish it were syntax-highlighted:

<img width="756" alt="image" src="https://user-images.githubusercontent.com/138052/157967748-43605e55-8878-40ae-90f7-426830720343.png">